bit_manipulation/swapping_two_numbers.cpp

```diff
@@ -0,0 +1,88 @@
+/**
+ * @file
+ * @brief Swapping two numbers using Bitwise operator
+ * More about it: https://betterexplained.com/articles/swap-two-variables-using-xor
+ * @details
+ * This program is part of bitwise manipulation programs, this one
+ * specifically helps us swap two numbers without using an auxillary
+ * variable.
+ * @author [Puneet Tripathi](https://github.com/PuneetTri)
+ */
+
+#include<iostream>  // For standard I/O, in our case printing values of variable 'a' and 'b'
+#include<cassert> // For assert
+
+bool bitwiseSwap(int a, int b) {
```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.